wire-runtime/src/main/java/com/squareup/wire/Wire.java

```diff
@@ -115,59 +115,6 @@ public Wire(List<Class<?>> extensionClasses) {
   }
 
   /**
-   * Reads a message of type {@code messageClass} from {@code bytes} and returns
-   * it.
-   */
-  public <M extends Message> M parseFrom(byte[] bytes, Class<M> messageClass) throws IOException {
```

Comment:
By who? This means someone is buffering all their data instead of using a stream.